astropy/io/fits/hdu/hdulist.py

```diff
@@ -510,6 +510,10 @@ def fileinfo(self, index):
 
         return output
 
+    def copy(self):
+        # Return a shallow copy when calling HDUList.copy() (issue #7211)
```

Comment:
This needs a real docstring instead of the comment (no need to put the issue number here).

Reply:
OK, I've changed it to represent a docstring.